wurst/_handles/Effect.wurst

```diff
@@ -76,16 +77,35 @@ public function effect.getLocalZ() returns real
 	return BlzGetLocalSpecialEffectZ(this)
 	
 public function effect.setPos(vec3 pos)
-	this.setPosition(pos.x, pos.y, pos.z)
-	
+	BlzSetSpecialEffectPosition(this, pos.x, pos.y, pos.z)
+
 public function effect.setPos(vec2 pos)
-	this.setPosition(pos.x, pos.y, this.getLocalZ())
+	BlzSetSpecialEffectPosition(this, pos.x, pos.y, BlzGetLocalSpecialEffectZ(this))
 
 public function effect.getPos3() returns vec3
-	return vec3(this.getLocalX(), this.getLocalY(), this.getLocalZ())
+	return vec3(BlzGetLocalSpecialEffectX(this), BlzGetLocalSpecialEffectY(this), BlzGetLocalSpecialEffectZ(this))
 	
 public function effect.getPos() returns vec2
-	return vec2(this.getLocalX(), this.getLocalY())
+	return vec2(BlzGetLocalSpecialEffectX(this), BlzGetLocalSpecialEffectY(this))
+
+public function effect.setColor(color color)
+	BlzSetSpecialEffectColor(this, color.red, color.green, color.blue)
+
+public function effect.setColor(colorA color)
+	BlzSetSpecialEffectColor(this, color.red, color.green, color.blue)
+	BlzSetSpecialEffectAlpha(this, color.alpha)
+
+public function effect.setOrientation(angle yaw, angle pitch, angle roll)
```

Comment:
IIRC with latest patch the yaw/pitch/roll functions still have issues and might produce unexpected behaviour. Ideally add some hotdoc explaining what it does (rotating around which axis, whether it works or not)

Reply:
I did a test with the yaw/pitch/roll. Based on my test, BlzSetSpecialEffectPitch is the only function having issues when it reaches around 90 to 270 degrees yet the pitch in BlzSetSpecialEffectOrientation seems to work fine. 

Should we keep BlzSetSpecialEffectPitch for setPitch and just add a warning? Or replace it with BlzSetSpecialEffectOrientation until the problem is fixed by Blizzard?

Reply:
Better set a warning I think